src/main/java/de/hysky/skyblocker/skyblock/profileviewer/ProfileViewerScreen.java

```diff
@@ -0,0 +1,243 @@
+                    return true;
+                }
+            };
+            entity.setCustomNameVisible(false);
+        }).exceptionally(ex -> {
+            this.playerName = "User not found";
+            return null;
+        });
+
+        return CompletableFuture.allOf(profileFuture, minecraftProfileFuture);
+    }
+
+    public void onNavButtonClick(ProfileViewerNavButton clickedButton) {
+        if (profileViewerPages[activePage] != null) profileViewerPages[activePage].markWidgetsAsInvisible();
+        for (ProfileViewerNavButton button : profileViewerNavButtons) {
+            button.setToggled(false);
+        }
+        activePage = clickedButton.getIndex();
+        clickedButton.setToggled(true);
+    }
+
+    @Override
+    public void init() {
+        profileViewerNavButtons.forEach(this::addDrawableChild);
+        for (ProfileViewerPage profileViewerPage : profileViewerPages) {
+            if (profileViewerPage != null && profileViewerPage.getButtons() != null) {
+                for (ClickableWidget button : profileViewerPage.getButtons()) {
+                    if (button != null) addDrawableChild(button);
+                }
+            }
+        }
+    }
+
+    public static void initClass() {
+        fetchCollectionsData(); // caching on launch
+        ClientCommandRegistrationCallback.EVENT.register((dispatcher, registryAccess) -> dispatcher.register(
+                ClientCommandManager.literal(SkyblockerMod.NAMESPACE)
+                        .then(ClientCommandManager.literal("pv")
+                                .then(ClientCommandManager.argument("username", StringArgumentType.string())
+                                        .executes(context -> {
+                                            String username = StringArgumentType.getString(context, "username");
+                                            Command<FabricClientCommandSource> cmd = Scheduler.queueOpenScreenCommand(() -> new ProfileViewerScreen(username));
+                                            return cmd.run(context);
+                                        })
+                                )
+                        )
+        ));
+        ClientCommandRegistrationCallback.EVENT.register((dispatcher, registryAccess) -> dispatcher.register(
+                        ClientCommandManager.literal("pv")
+                                .then(ClientCommandManager.argument("username", StringArgumentType.string())
+                                        .executes(context -> {
+                                            String username = StringArgumentType.getString(context, "username");
+                                            Command<FabricClientCommandSource> cmd = Scheduler.queueOpenScreenCommand(() -> new ProfileViewerScreen(username));
+                                            return cmd.run(context);
+                                        })
+                                )
+                )
+        );
```

Comment:
having it show your own profile when you just do `/pv` without any username would be nice